books/spiders/book_spider.py

```diff
@@ -0,0 +1,76 @@
+import scrapy
+from scrapy.http import Response
+
```

Comment:
It seems that you have imported the `Response` class from `scrapy.http` but you are not using it in your code. It's a good practice to avoid unnecessary imports to keep the code clean. Please remove it if it's not required.

Reply:
It's required, I use It!